static/src/lang/nl_NL/common.json

```diff
@@ -103,6 +103,8 @@
 		"cancel": "Annuleren",
 		"catalog": "Catalog",
 		"clickToCancel": "Click om te annuleren",
+		"cinemaStatusOneSpectatorTitle": "1 spectator is watching ",
```

Comment:
For these, perhaps use `cinemaStatusMultipleSpectatorsTitle` as `cinemaStatusSpectatorTitle` and use some small string manipulation to determine single or multiple instead of adding two separate entries?

Reply:
No, that might not be possible since different languages, my bad.